main.py

```diff
@@ -0,0 +1,6 @@
+from routes import app
+
+app.config["VERSION"] = "0.1.2"
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=80)
```

Comment:
Should it be port 80? Will this allow it to run on https?

Reply:
fuck knows tbf fam
Also how do u do Ur Https certs in your existing configuration?